firebase.json

```diff
@@ -315,6 +315,7 @@
       { "source": "/go/go-router-v6-breaking-changes", "destination": "https://docs.google.com/document/d/1CMJwd5Moq_scvHf-trRldy9RW5MU7_Y8IB1aTrny-6s/edit", "type": 301 },
       { "source": "/go/handling-synchronous-keyboard-events", "destination": "https://docs.google.com/document/d/1rWXSjkb2ZKv-cpg26lVK0aZi4cVeXJ8j7YmSJdq2TOM/edit", "type": 301 },
       { "source": "/go/hermetic-xcode-installation", "destination": "https://docs.google.com/document/d/1EcXm4Woq48GR_ky07mEJka6NfV1vFBN2OHDEeGfPk34/edit?resourcekey=0-0Gjtt1I66mGJ8AwiCTlFNw", "type": 301 },
+      { "source": "/go/hero-refactor", "destination": "https://docs.google.com/document/d/17EG-Sgp7kHVetX-odJ5jrgnpBWj2lx097sYtrSP5xno", "type": 301 },
```

Comment:
This doc doesn't seem to be public yet. Can you update it?

Reply:
It's not public for me either

Reply:
This is weird, I have to toggle the permission back and forth several times. This should be public now, I just test with another devices with non corp login

Reply:
Weird, I still can't access it.

Reply:
I changed to a different file. Let me know if this is still not working